after/plugin/dracula.vim

```diff
@@ -31,5 +31,10 @@ if exists('g:ale_enabled')
   hi! link ALEInfoSign DraculaCyan
 endif
 " }}}
+"}}}
+" CtrlP: {{{
+  hi! link CtrlPMatch Keyword
```

Comment:
2 quick things:

1. Can we add an `exists` check here to test for the existence of ctrlp before creating those highlight groups.
2. Can we switch from `Keyword` to `IncSearch` so there's better uniformity with other means of searching (FZF, builtin search, etc)

Reply:
Sounds good! I'll make those changes.

Reply:
Just pushed the requested changes.